lib/iris/experimental/ugrid/__init__.py

```diff
@@ -45,11 +45,11 @@
 from ...util import guess_coord_axis
 
 __all__ = [
-    "CFUGridReader",
     "Connectivity",
     "ConnectivityMetadata",
     "load_mesh",
     "load_meshes",
+    "save_mesh",
```

Comment:
@pp-mo I think this should come after `PARSE_UGRID_ON_LOAD`, but this is due to be refactored imminently, so not bothered either way